waspc/packages/deploy/src/providers/fly/createDb/CreateDbOptions.ts

```diff
@@ -0,0 +1,7 @@
+import { GlobalOptions } from '../GlobalOptions.js'
+
+export interface CreateDbOptions extends GlobalOptions {
```

Comment:
It would all basically be for the same end goal. We have some set of options shared by all commands. Then there are others that need those plus some that are specific to that command. This was just something I came up with. What would your types look like in your revision?

Reply:
Either CreateDbOptions would contain globalOptions as a field, or they would be passed around separately.
But you are right that end result is similar so probably best to leave it as it is.

Reply:
> We have some set of options shared by all commands.

This hints for me that the name should be `CommonOptions` vs. `GlobalOptions`

Reply:
Good point, `CommonOptions` is a more accurate name, will update it 👍🏻 Thanks!